src/app/features/management/user-management-view/edit-user-view/edit-user-view.component.spec.ts

```diff
@@ -0,0 +1,233 @@
+import {byTextContent, createRoutingFactory, SpectatorRouting, SpectatorRoutingOptions } from '@ngneat/spectator';
+import { NotificationService } from '../../../../core/services/notification.service';
+import { fakeAsync, tick } from '@angular/core/testing';
+import { of, throwError } from 'rxjs';
+import { UserService } from '../../../../core/services/user.service';
+import { EditUserView } from './edit-user-view.component';
+import { ManagementModule } from '../../management.module';
+
+function genFactoryOptions(): SpectatorRoutingOptions<EditUserView> {
+  return {
+    component: EditUserView,
+    imports: [
+      ManagementModule,
+    ],
+    mocks: [
+      NotificationService,
+      UserService,
+    ],
+    params: {
+      id: '007',
+    },
+    detectChanges: false,
+  };
+}
+
+describe('EditUserView', () => {
+  let spectator: SpectatorRouting<EditUserView>;
+  let component: EditUserView;
+  const createComponent = createRoutingFactory(genFactoryOptions());
+  const id = '007';
+  const username = 'JamesBond';
+  const firstname = 'James';
+  const lastname = 'Bond';
+  const isAdmin = false;
+  const isActive = true;
+
+  beforeEach(async () => {
+    spectator = createComponent();
+    component = spectator.component;
+    spectator.inject(UserService).getUserById.and.returnValue(of({
+      id: id,
+      username: username,
+      firstname: firstname,
+      lastname: lastname,
+      isAdmin: isAdmin,
+      isActive: isActive,
+    }));
+    spectator.router.navigate = jasmine.createSpy().and.callFake(spectator.router.navigate).and.resolveTo();
+    spectator.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+
+  it('should disable user update without values', () => {
+    expect(component.form.valid).toBeFalse();
+  });
+
+  it('should disable user update without username', fakeAsync(() => {
+    component.form.setValue({
+      username: '',
+      firstName: firstname,
+      lastName: lastname,
+      isAdmin: isAdmin,
+      isActive: isActive,
+    });
+    tick();
+    expect(component.form.valid).toBeFalse();
+  }));
+
+  it('should disable user update without firstname', fakeAsync(() => {
```

Comment:
minor typo with _firstname_ -> _first name_ and _lastname_ -> _last name_ (see other test cases as well)